### DISCUSSION TITLE
Project Starting File

### DISCUSSION THREAD

**Discussion by HarrisonHamer** [SimDynamX/SpaceTeamsPRO#11] [2023 Mars City Design Competition Q&A] [answered]
Howdy!

Is there a starting file with proper parameters provided, or do we create the file ourselves using New Sim? 
The Reference Demo does not show up in the Sim Config search so I'm not sure if that was it.

Thanks!

**Comment by connorjak** [comment, marked as answer]
Hi Harrison,

The Reference Sim is not included in the 0.20.0 space teams release. It will be included in the next release. For now, you can use the New Sim functionality or Solar System Explorer demo to explore Mars
